repository-data/webfiles/src/main/resources/site/scss/components/_navigation-tiles.scss

```diff
@@ -0,0 +1,138 @@
+.navigation-section {}
+
+.navigation-section__introduction {
+    margin-bottom: 20px;
+}
+
+.navigation-section--column .navigation-section__tiles {
+    @include media(desktop) {
+        display: flex;
```

Comment:
Have you validated this works in IE 11? It seems there is partial support for `display: flex` according to 'Can I Use' https://caniuse.com/#search=display%3A%20flex

Reply:
I actually wrote this some time ago, so I should probably circle back around and test it again. I use Flexbox in IE all the time, and often rely on https://github.com/philipwalton/flexbugs to solve small implementation-specific issues.

I'm 99% sure that `display: flex` works on IE11, and that the results on caniuse are due to a confusing table on MDN (where it gets it's source data from). But will double check!

Reply:
Here's a screenshot of it running in IE11, tested via Browserstack

![bs_win10_IE_11 0](https://user-images.githubusercontent.com/1337003/81723553-eb21c000-947a-11ea-9119-200b707832e4.jpg)